src/CharacterList.js

```diff
@@ -53,7 +53,7 @@ class CharacterList extends Component {
       <div>
         <h2 className="main_title">Characters</h2>
         <ul className="nav_items">
-          {alphabet.map((letter, index) => <AlphaLinks letter={letter} key={index} />)}
+          {alphabet.map((letter, index) => <AlphaLinks letter={letter} key={index} selected={this.state.letter} />)}
```

Comment:
I wonder if we should create a boolean to pass along as the selected prop? The effect is the same, just seems a little odd to me to pass a string and then check it in the component. Is selected={letter === this.state.letter} too messy?